packages/manager/modules/models/BillingService/BillingService.class.js

```diff
@@ -3,6 +3,7 @@ import head from 'lodash/head';
 import includes from 'lodash/includes';
 import isNull from 'lodash/isNull';
 import snakeCase from 'lodash/snakeCase';
+import 'moment';
```

Comment:
@cbourgois in lateral security position

Reply:
@FredericEspiau we could think on a refactoring for this model later on, here the purpose is just to allow reusability

Reply:
Yeah it wasn't a criticism, you are entirely right to do this